src/screens/ProfileScreen/index.js

```diff
@@ -51,6 +51,7 @@ import useProfileScreenHook, {
 } from '../../hooks/screen/useProfileScreenHook';
 import {Analytics} from '../../libraries/analytics/firebaseAnalytics';
 import {ButtonNewPost} from '../../components/Button';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `COLORS` from `../../utils/theme` has been moved to a different location in the file. This change doesn't seem to affect the functionality, but it's generally a good practice to keep all imports at the top of the file for better readability and maintainability.
```diff
- import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->